azurePipelineTemplates/run_scenarios.yml

```diff
@@ -0,0 +1,117 @@
+parameters:
+  - name: storage_account_name
+    type: string
+  - name: container_name
+    type: string
+  - name: orp_source_account_name
+    type: string
+  - name: orp_destination_account_name
+    type: string
+  - name: orp_source_container_name
+    type: string
+  - name: destination_container_name
+    type: string
+
+steps:
+  - task: AzureCLI@2
+    displayName: "Run Scenarios"
+    inputs:
+      azureSubscription: azcopyTestScenarios
+      addSpnToEnvironment: true
+      scriptType: pscore
+      scriptLocation: inlineScript
+      inlineScript: |
+        # Print "Building executable"
+        Write-Output "Building executable"
+        
+        # Set platform-specific environment variables and tags
+        $tags = ""
+        $suffix = ""
+        $build_name = ""
+        $display_name = ""
+        if ($IsWindows) {
```

Comment:
Based on the env you are setting env variables, not sure if this if-else is the best way to do this. We shall explore if we can do something better here.

Reply:
ok will change it